5.6/.sti/bin/assemble

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+echo "---> Installing application source ..."
+cp -Rf /tmp/src/. ./
+
+if [ -f composer.json ]; then
+  echo "Installing dependencies from composer.json... "
```

Comment:
Please make this message consistent with the message above :-) I know that I sounds like I nit picking, but in the end it makes the build logs in OpenShift look much nicer.